lib/gateway/Shard.js

```diff
@@ -2368,6 +2358,7 @@ class Shard extends EventEmitter {
                 }
                 const oldChannel = {
                     appliedTags: channel.appliedTags,
+                    autoArchiveDuration: channel.autoArchiveDuration,
                     flags: channel.flags,
                     name: channel.name,
                     rateLimitPerUser: channel.rateLimitPerUser,
```

Comment:
Missing archived, invitable, and locked props

Reply:
Sos assumed they were under threadMetadata